global/navigation@2.0.js

```diff
@@ -0,0 +1,51 @@
+!function () {
+  window.$loaded(function (window, document, $, undefined) {
+    $('.js-nav-mobile-menu-button').on('click', function (event) {
+      if ($('.js-nav-mobile-menu').hasClass('open-nav-menu')) {
+        $('.js-body').removeClass('no-scroll');
+        $('.js-primary-nav-bar').removeClass('fixed');
+        $('.js-nav-mobile-menu').removeClass('open-nav-menu');
+        $('.js-nav-mobile-menu').addClass('close-nav-menu');
+      } else {
+        $('.js-body').addClass('no-scroll');
+        $('.js-primary-nav-bar').addClass('fixed');
+        $('.js-nav-mobile-menu').removeClass('close-nav-menu');
+        $('.js-nav-mobile-menu').addClass('open-nav-menu').scrollTop(0);
+      }
+    });
+
+    var ready = function () {
+      (function trackAnalytics() {
+        $('[data-js="track-nav"]').each(function () {
```

Comment:
@jethroflanagan On some pages, the data attribute _data-js_ is set to _**track-data**_ instead because this same js is then used to track analytics for certain CTAs. 

I wonder if we should create a file for tracking other call to actions separately where we set the _data-js_ as _**track-data**_ or just amend this one to accommodate for both scenarios (aka _track-nav_ or _track-data_)

Reply:
@chama-ram I don't fully understand how it works, let's catchup and then decide.